### ISSUE TITLE
Tests may fail with custom ~/.gdbinit

### ISSUE THREAD

**Issue by bchretien** [skibyte/gdblib#14]
If the user has a custom `~/.gdbinit` file (e.g. [for customization](http://stackoverflow.com/a/17341335/1043187)), the tests may fail since the expected behavior could change. In my case, the default printing style is changed in gdb, leading to errors in the tests.

Command:

``` sh
$ python2 setup.py test
```

Output for the print tests:

``` txt
======================================================================
FAIL: testPrint (gdblib.gdbtestcase.GDBTestCase)
----------------------------------------------------------------------
Traceback (most recent call last):
  File ".../python2-gdblib/src/gdblib-0.5/gdblib/gdbtestcase.py", line 187, in testPrint
    self.assertEquals('$1 = 6', self.connectedGdb.p('mypoint.x'))
AssertionError: '$1 = 6' != '$1 = 0x6'

======================================================================
FAIL: testPrint_Incorrect (gdblib.gdbtestcase.GDBTestCase)
----------------------------------------------------------------------
Traceback (most recent call last):
  File ".../python2-gdblib/src/gdblib-0.5/gdblib/gdbtestcase.py", line 195, in testPrint_Incorrect
    self.connectedGdb.p('e'))
AssertionError: 'No symbol \\"e\\" in current context.\\n' != '$1 = 0xe'

----------------------------------------------------------------------
Ran 62 tests in 34.134s

FAILED (failures=2)
```

This may be something that needs to be handled in gdblib (e.g. the `-nh/-n` options of GDB that prevents custom initialization files to be loaded, at least for the tests).

**Comment by skibyte** [comment]
Thanks I will handle this during the next week.

Best Regards.

**Comment by skibyte** [comment]
Hi could you please share your gdbinit file? I am not able to reproduce the problem.

Regards.

**Comment by bchretien** [comment]
They're available [here](https://github.com/bchretien/Gdbinit).

**Comment by skibyte** [comment]
Fixed by 70c8aa30c04f

**Comment by bchretien** [comment]
:+1: Thanks!
